graphics/asymptote/files/patch-Makefile.diff

```diff
@@ -1,38 +1,20 @@
---- Makefile.in.orig
-+++ Makefile.in
-@@ -10,7 +10,7 @@ LIBATOMIC = libatomic_ops-@ATOMICVERSION@
+--- Makefile.in.orig	2022-05-14 12:20:39.000000000 +0700
++++ Makefile.in	2022-08-16 19:46:32.000000000 +0700
+@@ -10,7 +10,7 @@
  GCOPTIONS = @GCOPTIONS@
  GCLIB = @GCLIB@
  GCPPLIB = @GCPPLIB@
 -GCLIBS = $(GCPPLIB) $(GCLIB)
 +GCLIBS = $(GCPPLIB) $(GCLIB) -lgccpp
- LFLAGS = @LDFLAGS@
- LIBS = $(LFLAGS) @PTHREAD_LIBS@ @GLEW@ @LIBS@ $(GCLIBS)
- DOSLIBS = $(subst -lncurses, -ltermcap, $(LIBS)) -lgdi32 -lwinmm -s -static
-#@@ -110,7 +110,7 @@ endif
-# export prefix docdir exampledir mandir infodir INSTALL MAKE DESTDIR TEXI2DVI
-# 
-# asy:	version
-#-	-pyrcc5 GUI/res/icons.qrc -o GUI/icons_rc.py
-#+	$(PYRCC5) GUI/res/icons.qrc -o GUI/icons_rc.py
-# 	if test -n "$(MSDOS)"; then \
-#           $(CXX) $(OPTS) -o $(NAME) $(FILES:=.o) revision.o asy.o $(DOSLIBS); \
-# 	else \
-@@ -137,7 +137,7 @@ version: $(GCLIB) $(FILES:=.o) $(UIFILES:.ui=.py)
- 	echo @set VERSION $(revision) > doc/version.texi
+ LFLAGS = @LDFLAGS@ -LLspCpp -LLspCpp/third_party/uri/src
```

Comment:
Thanks a lot, I'll be testing this now. I wasn't able to get LSP to work, so I'll check how this performs.

Reply:
I see you only adapted the patch to conform to the latest version in upstream sources rather than making some significant change.

The code still doesn't build for me (macOS 11, arm64).

Reply:
> I see you only adapted the patch to conform to the latest version in upstream sources rather than making some significant change.
> 
> The code still doesn't build for me (macOS 11, arm64).

This seems to be working as is with `gcc12` (but failing direly with `gcc-4.2`) – until doc generating. I do not know what goes so horribly wrong at that stage.

Reply:
UPD. Now everything should build.